main.c

```diff
@@ -0,0 +1,26 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+int main(void)
+{
+	char str[20];

```

Comment:
Seems you used tab instead of whitespaces as it was used in the previous PR.

Reply:
Done